projects/demo/src/modules/app/home/examples/app-standalone.md

```diff
@@ -0,0 +1,16 @@
+```ts
+import {Component} from '@angular/core';
+import {TuiRootModule} from '@taiga-ui/core';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./my.component.scss', './my.component.less'],
+  standalone: true,
+  imports: [
+    TuiRootModule,
```

Comment:
`TuiRootComponent`

Reply:
no, it seems we need to import `TuiRootModule` here while `TuiRootComponent` is not standalone